### PR TITLE
Add composer and selenium instructions to the README

### PR DESCRIPTION
**Pull request by philwinkle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,40 @@
 
 Magento Testing Framework (MTF) is an open source R&D project facilitating the improvement of Magento Core products. Magento Testing Framework helps in avoiding duplication of test cases automated across the application utilizing data driven approach. With the new and improved Magento Testing Framework, developers will be able to automate functional testing of the developed functionality enabling frequent releases and higher product quality.
 
+# How to Use
+
+If you are using MTF in your existing Magento Community 1.9.2+ or Magento Enterprise 1.14.2+ repositories you will find it located within the `/dev` folder in your Magento Root directory.
```
Comment (lanken) [inline README.md]: MTF is used not only for Magento 1 but for Magento 2 as well, paths are the same.

Comment (lanken) [inline README.md]: Actually functional tests are located at /dev/tests/functional

File: README.md [hunk 2]
```diff
@@ -2,6 +2,40 @@
...
+
+This will interpret tests from the directory of tests itself. The configuration file it will automatically read from is `phpunit.xml.dist` located within the `/dev/tests/functional` directory.
+
+Selenium Server
+--
+
+Functional tests are run with Selenium Server as a standalone JAR file. To execute this you need to issue this command:
+
+```bash
+java -jar vendor/netwing/selenium-server-standalone/selenium-server-standalone-2.46.0.jar
```
Comment (lanken) [inline README.md]: With the latest releases we don't download selenium server using composer, only suggest it.
Also it's better to specify some abstract version in documentation.

**Comment by lanken** [comment]
@philwinkle, thanks for your contribution.
I've left some comments to your pull request.

BTW, there are also some configuration files:
https://github.com/magento/mtf/blob/develop/docs/install-config.md
https://github.com/magento/mtf/blob/develop/docs/running.md

**Comment by lanken** [comment]
@philwinkle, as there is on response we updated README file by ourselves.
Please, check it out and reopen a ticket in case if README still requires changes.

**Comment by philwinkle** [comment]
That's great. Thanks.

On Thu, Oct 1, 2015 at 7:20 AM, Dmytro Aponasenko <notifications@github.com>
wrote:

> Closed #11 <https://github.com/magento/mtf/pull/11>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/magento/mtf/pull/11#event-424115652>.
>



-- 

Follow me on Twitter - @philwinkle
Friend me on Facebook - http://www.facebook.com/philwinkle

**Pull request closed without merging**